src/BlazorWebView/src/Maui/MauiBlazorWebViewBuilder.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Components.WebView.Maui
+{
+	internal class MauiBlazorWebViewBuilder : IMauiBlazorWebViewBuilder
+	{
+		/// <inheritdoc/>
+		public IServiceCollection Services { get; }
+
+		internal MauiBlazorWebViewBuilder(IServiceCollection services)
```

Comment:
This can be `public` since the class is `internal`. No XML docs are needed within the internal class.